pkg/monitor/dynkeepalived.go

```diff
@@ -70,6 +71,26 @@ func updateUnicastConfig(kubeconfigPath string, newConfig, appliedConfig *config
 	if err != nil {
 		log.Warnf("Could not retrieve LB config: %v", err)
 	}
+	if os.Getenv("IS_BOOTSTRAP") == "yes" {
+
+		for _, value := range serverBackends {
+			ipExist := false
+			for _, lbValue := range newConfig.LBConfig.Backends {
+				if value.Address == lbValue.Address {
+					ipExist = true
+				}
+			}
+			if !ipExist {
+				newConfig.LBConfig.Backends = append(newConfig.LBConfig.Backends, config.Backend{Host: "", Address: value.Address})
```

Comment:
Instead of managing ipExist, isn't it the same to put this block inside line 80 (instead of ipExist=True) ?

Reply:
Hmm, but we need to add it to the backends only if it doesn't exist.
Only after we scan all the array and didn't find it.